### PR TITLE
Update AlertManager configuration page for AM v0.18

### PR DESCRIPTION
**Pull request by simonpasquier**
Follow-up of https://github.com/prometheus/alertmanager/pull/1863. It should be merged only after AlertManager v0.18 is released. cc @bwplotka

### CODE REVIEW

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -603,6 +604,18 @@ OpsGenie notifications are sent via the [OpsGenie API](https://docs.opsgenie.com
 [ http_config: <http_config> | default = global.http_config ]
 ```
 
+### `<responder>`
+
+```yaml
+# One of these fields should be defined.
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: Exactly or at least?

Comment (simonpasquier) [reply content/docs/alerting/configuration.md]: I think it's "exactly" but @bwplotka could confirm.

Comment (bwplotka) [reply content/docs/alerting/configuration.md]: They say nothing about this stricly so I would assume `exactly.
```
 Either id or name of each responder should be provided.You can refer below for example values.
```

**Comment by simonpasquier** [comment]
@brian-brazil ok to merge?

**Comment by simonpasquier** [comment]
Thanks!

**Pull request merged**